tools/decode-packets/src/main/scala/DecodePackets.scala

```diff
@@ -0,0 +1,165 @@
+import java.io.{File, ByteArrayOutputStream, FileOutputStream}
+import java.nio.charset.CodingErrorAction
+import net.psforever.packet.PacketCoding
+import scodec.bits._
+import scodec.Attempt.{Failure, Successful}
+import java.nio.file.{Files, Paths, StandardCopyOption}
+import scala.io.Codec
+import util.control.Breaks._
+import scala.collection.parallel.CollectionConverters._
+import scopt.OParser
+import sys.process._
+
+case class Config(
+    outDir: String = System.getProperty("user.dir"),
+    files: Seq[File] = Seq()
+)
+
+object DecodePackets {
+  def main(args: Array[String]): Unit = {
+
+    val builder = OParser.builder[Config]
+
+    val parser = {
+      import builder._
+      OParser.sequence(
+        programName("psf-decode-packets"),
+        opt[String]('o', "out-dir")
+          .action((x, c) => c.copy(outDir = x))
+          .text("Output directory"),
+        arg[File]("<file>...")
```

Comment:
Does this work if you pass in a folder instead of individual files? I'd rather not list 655 files in one argument 😄

Reply:
`*.gcap`? The shell should handle that just fine. Also more flexible since you can do stuff like `**/*.gcap` to convert everything in subfolders.

Reply:
I guess the question there is does file globbing work in the various ways to run it? Could be run through sbt shell directly, a terminal (windows, or linux) or through IDEA

Reply:
Linux and Windows terminal all support globs. No idea if it works in IDEA, if it runs stuff in a terminal then it should. Sbt console is a Scala REPL, so no, I don't think so. The idea was to just install it to your PATH so you can call it via the terminal from anywhere.